src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import { Error } from '../../types/Error';
+
+type Props = {
+  error: Error | null;
+  setError: (value: null) => void;
+  setIsLoading: (value: boolean) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  error,
```

Comment:
```suggestion
  error, // Is this an object? rename it)
```
[error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) as an object